frontend/src/components/Forms/AddPermitAmendmentForm.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Field, reduxForm } from "redux-form";
+import RenderDate from "@/components/common/RenderDate";
+import RenderAutoSizeField from "@/components/common/RenderAutoSizeField";
+import { Form, Button, Col, Row, Popconfirm } from "antd";
+import * as FORM from "@/constants/forms";
+import { required, maxLength, dateNotInFuture } from "@/utils/Validate";
+import { resetForm } from "@/utils/helpers";
+
+const propTypes = {
+  handleSubmit: PropTypes.func.isRequired,
+  closeModal: PropTypes.func.isRequired,
+  title: PropTypes.string.isRequired,
+  submitting: PropTypes.bool.isRequired,
+};
+
+export const AddPermitAmendmentForm = (props) => (
+  <Form layout="vertical" onSubmit={props.handleSubmit}>
+    <Row gutter={16}>
+      <Col>
+        <Form.Item>
+          <Field
+            id="issue_date"
+            name="issue_date"
+            label="Issue date"
+            component={RenderDate}
+            validate={[required, dateNotInFuture]}
+          />
+        </Form.Item>
+        {props.initialValues.permit_amendment_type_code !== "OGP" && (
```

Comment:
Not a blocking change, but something I did in the Contact advanced search was put a similar condition like this into a function so that the logic was a bit more explicit (via a variable name -- `isPerson`, in that case).

Reason being, it's not clear to me as a reader what "OGP" means

Reply:
Good practice. Now reads "originalPermit".